charactersheet/charactersheet/templates/dm/encounter_detail.tmpl.html

```diff
@@ -20,6 +20,7 @@
         templateUrl: 'templates/dm/encounter_sections',
         data: $parent[section.property]()
       }"></div>
+      <hr />
```

Comment:
`<hr/>` tags are always visible, even when the section isn't displayed. Please fix so that they're conditional on the section. 
![image](https://cloud.githubusercontent.com/assets/7286387/20916247/9c907372-bb3f-11e6-9506-6417f3b85088.png)